src/client/Microsoft.Identity.Client/Internal/Logger/MsalLogger.cs

```diff
@@ -192,7 +192,19 @@ public void Log(LogLevel logLevel, string messageWithPii, string messageScrubbed
                 bool isLoggingPii = messageWithPiiExists && PiiLoggingEnabled;
                 string messageToLog = isLoggingPii ? messageWithPii : messageScrubbed;
 
-                string log = $"{isLoggingPii} MSAL {s_msalVersionLazy.Value} {s_skuLazy.Value} {s_runtimeVersionLazy.Value} {s_osLazy.Value} [{DateTime.UtcNow:MM/dd HH:mm:ss.ff}{_correlationId}]{ClientInformation} {messageToLog}";
+                string log = string.Format(
+                    CultureInfo.InvariantCulture,
+                    "{0} MSAL {1} {2} {3} {4} [{5}{6}]{7} {8}",
```

Comment:
Would people actually search by the date in this log string, since this string would have to be parsed? In Kusto, this string goes into a message column, and then there's a separate date time column, which is used for filtering I think.

Reply:
Along with that, if the date is parsed out of this whole log string usually, then changing it would break people's current parsing logic?

Reply:
If you use LogsMiner, they expect the data to be in this format. Also AAD sends timestamp in this format as well. 
I understand that some ppl might parse the logs, but we don't gurantee backwards compat at this level.

 There is an ask to provide a structured log, which we should investigate as part of the logging improvements issue